bcda/web/api_test.go

```diff
@@ -1178,16 +1178,6 @@ func (s *APITestSuite) TestJobStatusFailed() {
 
 	assert.Equal(s.T(), http.StatusInternalServerError, s.rr.Code)
 
-	var oo fhirmodels.OperationOutcome
```

Comment:
Hmm, do you need to pull `master` into this branch?  Isn't this from a previous PR?